pkg/portal/portal.go

```diff
@@ -136,12 +137,22 @@ func (p *portal) setupRouter() error {
 	r := mux.NewRouter()
 	r.Use(middleware.Panic(p.log))
 
-	asset, err := Asset("index.html")
+	assetv1, err := Asset("v1/dist/index.html")
 	if err != nil {
 		return err
 	}
 
-	p.t, err = template.New("index.html").Parse(string(asset))
+	assetv2, err := Asset("v2/build/index.html")
+	if err != nil {
+		return err
+	}
+
+	p.tempv1, err = template.New("index.html").Parse(string(assetv1))
```

Comment:
nit picking here, but `temp` usually makes me believe we are working with short lived temporary vars to hold intermediate states between operations. It looks like in this context we are referring to templates with different versions. it is a little confusing.

Reply:
Absolutely. I will adjust the name here

Reply:
updated to template